### DISCUSSION TITLE
Mokura: Option to disable nagivation per swipe

### DISCUSSION THREAD

**Discussion by Vhalesa** [arianneorpilla/jidoujisho#312] [Q&A]
Hello,

would it be possible to add an option to completely turn off the page navigation per swipe in the mokuro viewer? So that you'll just use the buttons to navigate? (Not sure, if that's even an option more people would want, but I always have to be very careful not to turn the page when I zoom in and move around)

**Comment by brishtibheja** [comment]
> Not sure, if that's even an option more people would want, but I always have to be very careful not to turn the page when I zoom in and move around

yeah faced this problem especially with double spreads that you always have to zoom in on phone. iirc we will get an update to the mokuro reader(not very soon) that will integrate the reader here: https://reader.mokuro.app/

you can also use this until then with yomichan on kiwi browser.

**Reply by Vhalesa** [reply]
Ohhh, perfect! 
ty for the quick reply and the hard work :D
